docs/source/hardware/low_priority.md

```diff
@@ -0,0 +1,57 @@
+(low-priority)=
+
+# Low Priority Partition
+
+## Introduction
+
+`lowpriority` is a new partition on our high performance computing (HPC) cluster, Discovery which allows the research community to use private resources on the HPC cluster when they are idle. This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. This new partition has hardware that is not otherwise available to the general research community and, in time, could double the resources available to NURC users.
+
+## When to use the Low Priority Partition
+
+From most to least-recommended scenarios:
+
+1. Code that can be checkpointed (Please see {ref}`checkpoint-jobs`)
+1. Jobs that fit on a single node 
+1. Jobs that require multiple nodes (e.g., MPI - please see {ref}`using-mpi`)
+1. When waiting for your jobs to get started is too hard
```

Comment:
Maybe rephrase to: When facing longer than average waits times in the other partitions